src/plugins/tools/apipTextToSpeech/plugin.js

```diff
@@ -45,6 +46,24 @@ export default pluginFactory({
         const testRunner = this.getTestRunner();
         const testRunnerOptions = testRunner.getOptions();
         const pluginShortcuts = (testRunnerOptions.shortcuts || {})[this.getName()] || {};
+        let ttsComponent = null;
+
+        /**
+         * Creates the tts component on demand
+         * @returns {textToSpeech}
+         */
+        const getTTSComponent = () => {
+            if (!ttsComponent) {
+                const $container = testRunner.getAreaBroker().getContainer();
+
+                ttsComponent = ttsComponentFactory($container.parent(), {})
```

Comment:
Is it required to place the component on the container's parent? It is unsafe, and it should be safer to not place it higher than the container.